src/main/kotlin/day_1_kotlin/assignments/KotlinWorkshop6.kt

```diff
@@ -1,34 +1,48 @@
 package day_1_kotlin.assignments
 
-// Workshop #5 - special classes, objects, data classes
+// Workshop #6 - special classes, objects, data classes
 
-// TODO 1: add 2-4 fields to the data class
-// data class VideoGame()
 
-// TODO 2: create on object that will be used to test the games, call it "VideoGamesTest"
+
+/* Exercise area */
+
+// TODO 1: Add 2-4 properties to the data class
+// data class VideoGame(...)
+
+// TODO 2: Create on object that will be used to test the games, call it "VideoGamesTest".
 // ... {
 
-      // Uncomment after declaring an object
+      // Uncomment after declaring an object.
 //    @JvmStatic
 //    fun main(args: Array<String>)  {
-        // TODO 3: create an instance of VideoGame, make a copy via .copy() method and
-        //  put the copy to another variable
+        // TODO 3: Create an instance of VideoGame.
+        //  Make a copy via .copy() method and store the copy to another variable.
         // val game =
         // val copy =
 
-        // TODO 4: print out if those data class objects are equal (use "==" for comparison). Should be true
+        // TODO 4: Print out if those data class objects are equal.
+        //  Use "==" to compare. Should be true.
 //        val equal = false
 //        println("Objects are equal $equal")
 
-        // TODO 5: put several data class objects into an array and print them out
+        // TODO 5: Put several data class objects into an array and print them out.
 //        val games = arrayOf(game, copy, ...)
 //        games.forEach {
 //            // print
 //        }
 //    }
 //}
 
-// TODO 6: create an enum Genre with a few values, and add it as a field to the VideoGame
-
-// * * * Bonus task * * * (work on home)
-// Create a new class hierarchy for an arbitrary topic
\ No newline at end of file
+// TODO 6: Create an enum class Genre with a few values, and add it as a field to the VideoGame.
```

Comment:
Looks like this one should start the bonus section

Reply:
Fixed with ammend commit.